azure-pipelines-ci.yml

```diff
@@ -21,6 +21,11 @@ resources:
 extends:
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32050&commitId=66fd454865e5e197ae4343af254d93125dd3bfa3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Selectable</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>